core/application/impl/chain_spec_impl.cpp

```diff
@@ -162,16 +163,37 @@ namespace kagome::application {
     if (code_substitutes_opt.has_value()) {
       for (const auto &[hash, code] : code_substitutes_opt.value()) {
         OUTCOME_TRY(hash_processed, common::Hash256::fromHexWithPrefix(hash));
-        OUTCOME_TRY(code_processed, common::unhexWith0x(code.data()));
-        // TODO(sanblch): move this from memory to db
-        // https://github.com/soramitsu/kagome/issues/935
-        code_substitutes_->emplace(hash_processed, code_processed);
+        // OUTCOME_TRY(code_processed, common::unhexWith0x(code.data()));
+        code_substitutes_->emplace(hash_processed);
       }
     }
 
     return outcome::success();
   }
 
+  outcome::result<common::Buffer> ChainSpecImpl::fetchCodeByHash(const common::Hash256 hash) const {
```

Comment:
Better to name `fetchCodeSubstituteByHash`. Code could be confusing as there is another code in chain spec